extension.neon

```diff
@@ -119,6 +119,11 @@ services:
 		arguments:
 			managerClass: Doctrine\Common\Persistence\ObjectManager
 
+	dbalTypeGetType:
```

Comment:
This service can be marked only with `-` (because it's very unlikely someone will want to override it) and moved above to other anonymous `-` services.